app/components/record_actions/view.rb

```diff
@@ -53,7 +53,7 @@ def defer_link
     end
 
     def withdraw_link
-      govuk_link_to(t("views.trainees.edit.withdraw"), trainee_withdrawal_path(trainee), class: "withdraw")
+      govuk_link_to(t("views.trainees.edit.withdraw"), trainee_start_date_verification_path(trainee, context: :withdraw), class: "withdraw")
```

Comment:
I think the delete link above might need to be the following:

```ruby
    def delete_link
      govuk_link_to(t("views.trainees.edit.delete"), trainee_start_date_verification_path(trainee, context: "delete"), class: "delete")
    end
```

Reply:
thanks, fixed